tests/examples/awful/tooltip/gaps.lua

```diff
@@ -0,0 +1,30 @@
+--DOC_GEN_IMAGE
+--DOC_NO_USAGE
+--DOC_HIDE_ALL
+screen[1]._resize {width = 300, height = 150}
+screen.no_outline = true
+local awful = {tooltip = require("awful.tooltip")}
+local beautiful = require("beautiful")
+local wibox = require("wibox")
+
+-- mouse.coords{x=50, y= 10}
+
+    local wb = wibox {width = 100, height = 44, x = 100, y = 50, visible = true}
+
+require("gears.timer").run_delayed_calls_now()
+
+for _, side in ipairs{ "left", "right", "bottom", "top" } do
+    local tt = awful.tooltip {
+        text = side,
+        objects = {wb},
+        mode = "outside",
+        preferred_positions = {side},
+        preferred_alignments = {"back"},
+        tooltip_gaps = 5
```

Comment:
It's called `gaps` now, right?

Reply:
Yeah haha. Anyway, what about the theme variable, `beautiful.tooltip_gaps`? Should we stay with that name? I feel that `beautiful.gaps` is a bad name for that variable.

Reply:
> beautiful.tooltip_gaps

Yes, that's correct. Beautiful "naming convention" is to go from left to right with module prefix. So in this case, `tooltip_` is the prefix. Some properties also have suffixes for the state.

Another example would be `bg`. You have `beautiful.tasklist_fg_focus` and `beautiful.tasklist_fg_normal`. They fallback to `beautiful.bg_focus` and `beautiful.bg_normal` respectively. This is a good example of the namespace/module prefix + state suffix is applied.